__tests__/features/index-page.feature

```diff
@@ -4,10 +4,15 @@ Feature: Index page
   I want to visit the index page
   So that I can start using the service
 
-  Scenario: Page has content
+  Scenario: Page is the index page
     When I visit /
-    Then the page title should be "Tenancy & Household Check - Index"
-    And I should see "This will be a start page!" on the page
+    Then the page title should be "Tenancy & Household Check"
+    And I should see "Please wait until the 'Go' button" on the page
```

Comment:
Didn't we change this to be a heading or something?

Reply:
Do you mean the title for the remultiform pages? Or I may be misunderstanding

Reply:
I mean to test for the presence of a heading, rather than some random body test that's likely to change.

Reply:
Changed to the title of the page, which just so happens to be 'Tenancy & Household Check' also